drivers/clock_control/Kconfig

```diff
@@ -23,6 +30,8 @@ module = CLOCK_CONTROL
 module-str = clock control
 source "subsys/logging/Kconfig.template.log_config"
 
+
+if !CLOCK_CONTROL_MUST_USE_OUT_OF_TREE_DRIVER
```

Comment:
Bit unbalanced to have no blank line after the `if` but a blank line before the `endif`.

I like blank lines for both. Less scrunched up.

Reply:
I'm usually a stickler for consistency. I can fix this and let the config breathe.